mon-pix/app/components/navbar-mobile-header.js

```diff
@@ -1,16 +1,9 @@
-import { classNames } from '@ember-decorators/component';
 import { inject as service } from '@ember/service';
 import { alias } from '@ember/object/computed';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
+import Component from '@glimmer/component';
 
-@classic
-@classNames('navbar-mobile-header')
 export default class NavbarMobileHeader extends Component {
   @service session;
 
-  @alias('session.isAuthenticated')
-  isUserLogged;
-
-  burger = null;
+  @alias('session.isAuthenticated') isUserLogged;
```

Comment:
Le décorateur alias est un computed. 

La version glimmer complète doit prendre cette forme il me semble
```suggestion
  get isUserLogged() {
  	return this.session.isAuthenticated;
  }
```

Reply:
Mais dans ces examples ici https://guides.emberjs.com/release/upgrading/current-edition/native-classes/, ils ont gardé le décorateur `alias`

Reply:
En lisant le document, je penche du côté de @celineung, utiliser l'alias semble encore d'actualité.